github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -19,25 +19,31 @@ message WrapperConfig {
 // decryption in an envelope fashion
 message EnvelopeInfo {
   // Ciphertext is the ciphertext from the envelope
+  // @gotags: `class:"public"`
```

Comment:
Nit: I found that the backticks are optional, and I kinda prefer this without them? In Boundary we're not using either consistently, but I think we can do it here .
```suggestion
  // @gotags: class:"public"
```

Reply:
ty